### PR TITLE
Simple explorer

### PR DESCRIPTION
**Pull request by ineiti**
Adds an explorer to omniledger with links to other instances.

Instead of waiting for the full phonebook, already merging this now, so that it can be used with an upcoming update for the personhood-contract.

### CODE REVIEW

File: dynacred/src/Config.ts [hunk 1]
```diff
@@ -33,6 +33,13 @@ export class Config {
             return Buffer.from(field, "hex");
         };
 
+        const asBool = (field: any): boolean => {
```
Comment (tharvik) [inline dynacred/src/Config.ts]: Hum, TOML supports boolean directly

File: webapp/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,13 +1,14 @@
 import { CommonModule } from "@angular/common";
 import { NgModule } from "@angular/core";
 import { RouterModule, Routes } from "@angular/router";
+import { ExplorerComponent } from "src/app/explorer/explorer.component";
```
Comment (tharvik) [inline webapp/src/app/app-routing.module.ts]:
```suggestion
import { ExplorerComponent } from "./explorer/explorer.component";
```

File: webapp/src/app/explorer/darc/darc.component.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+type ExprValue = [string, string];
+type Rule = [string, ExprValue[]];
+
+@Component({
+  selector: "app-darc",
+  templateUrl: "./darc.component.html",
+})
+export class DarcComponent implements OnChanges {
+
+  rules: Rule[];
```
Comment (tharvik) [inline webapp/src/app/explorer/darc/darc.component.ts]:
```suggestion
  rules: Rule[] | undefined;
```

File: webapp/src/app/explorer/explorer.component.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+    constructor(
+        private readonly route: ActivatedRoute,
+        private readonly uData: UserData,
+    ) {}
+
+    async ngOnInit() {
+        this.route.paramMap.subscribe(async (params) => {
+            Log.print("new params:", params);
+            Log.print("new params:", params.get("id"));
```
Comment (tharvik) [inline webapp/src/app/explorer/explorer.component.ts]: debug?

File: dynacred/src/Config.ts (hunk 1 above)
Comment (ineiti) [reply dynacred/src/Config.ts]: debugging...

**Pull request merged**